application/src/main/java/org/togetherjava/tjbot/commands/SlashCommand.java

```diff
@@ -110,78 +86,26 @@
      * in a very specific format, otherwise the core system will fail to identify the command that
      * corresponded to the button or menu click event and is unable to route it back.
      * <p>
-     * The component ID has to be a UUID-string (see {@link java.util.UUID}), which is associated to
-     * a specific database entry, containing meta information about the command being executed. Such
-     * a database entry can be created and a UUID be obtained by using
-     * {@link ComponentIdGenerator#generate(ComponentId, Lifespan)}, as provided by the instance
-     * given to {@link #acceptComponentIdGenerator(ComponentIdGenerator)} during system setup. The
-     * required {@link ComponentId} instance accepts optional extra arguments, which, if provided,
-     * can be picked up during the corresponding event (see
-     * {@link #onButtonClick(ButtonInteractionEvent, List)},
-     * {@link #onSelectionMenu(SelectMenuInteractionEvent, List)}).
-     * <p>
-     * Alternatively, if {@link SlashCommandAdapter} has been extended, it also offers a handy
-     * {@link SlashCommandAdapter#generateComponentId(String...)} method to ease the flow.
-     * <p>
-     * See <a href="https://github.com/Together-Java/TJ-Bot/wiki/Component-IDs">Component-IDs</a> on
-     * our Wiki for more details and examples of how to use component IDs.
-     * <p>
-     * This method will be called in a multi-threaded context and the event may not be hold valid
-     * forever.
+     * See {@link #acceptComponentIdGenerator(ComponentIdGenerator)} for more info on the ID's
      *
      * @param event the event that triggered this
      */
     void onSlashCommand(@NotNull SlashCommandInteractionEvent event);
 
     /**
-     * Triggered by the core system when a button corresponding to this implementation (based on
-     * {@link #getData()}) has been clicked.
+     * Triggered by the core system when a slash command's autocomplete corresponding to this
+     * implementation (based on {@link #getData()}) has been triggered.
```

Comment:
What is an autocomplete event? Maybe you have to write an extra sentence to that.

Reply:
We don't explain buttons/selection menus either. So I'll explain all of them then

Reply:
awesome, thank you